fury/ui.py

```diff
@@ -1100,11 +1100,20 @@ def __init__(self, text="Text Block", font_size=18, font_family='Arial',
             Makes text italicised.
         shadow : bool
             Adds text shadow.
+        border : bool
+            show bounding box.
+        bbox_size : (int, int)
+            width & height of bounding box
```

Comment:
you can remove the comment above and and size

Reply:
sure